feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -171,12 +171,6 @@ fun PreviewScreen(
                         modifier = Modifier
                             .fillMaxSize(),
                         onSurfaceProviderReady = onSurfaceProviderReady,
-                        onRequestBitmapReady = {
```

Comment:
Is this also functionality that we'll be losing by migrating? Do we need to request a feature for this?